crates/wash-lib/src/cli/registry.rs

```diff
@@ -63,11 +63,11 @@ pub struct RegistryPullCommand {
 #[derive(Parser, Debug, Clone)]
 pub struct RegistryPushCommand {
     /// URL to push artifact to
-    #[clap(name = "url")]
-    pub url: String,
+    #[clap(name = "url", env = "WASH_REG_URL")]
+    pub url: Option<String>,
 
     /// Path to artifact to push
-    #[clap(name = "artifact")]
+    #[clap(long = "artifact")]
```

Comment:
Since `artifact` is always required, doesn't it make more sense to keep this as a positional argument?